server/app/views/questiontypes/CurrencyQuestionFragment.html

```diff
@@ -12,10 +17,16 @@
 
   <!--/* Display errors */-->
   <div
-    th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${currencyQuestion.getValidationErrors().get(question.getContextualizedPath())})}"
+    th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${questionErrors})}"
+  ></div>
+  <div
+    th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${fieldErrors})}"
   ></div>
 
-  <div class="usa-input-group">
+  <div
+    class="usa-input-group"
```

Comment:
question (non-blocking):  Does a currency question ever have an input group?

Reply:
the reason the input group styling is there is that the "$" that comes prefilled in the left makes the styling slightly different. Without it the "$" won't render inside the input